packages/core/src/model.ts

```diff
@@ -433,7 +433,7 @@ export class Model implements Storage {
   }
 
   async find<T extends Doc> (clazz: Ref<Class<T>>, query: AnyLayout): Promise<T[]> {
-    return this.findSync(clazz, query) as T[]
+    return (await this.findSync(clazz, query)) as T[]
```

Comment:
`findSync` name implies it is NOT an async function, so it shall not return `Promise`, why we `await` here?

Reply:
I was confused by eslint :-)